{{cookiecutter.project_slug}}/.github/workflows/ci.yml

```diff
@@ -28,13 +28,16 @@ jobs:
         with:
           python-version: 3.8
 
-      - name: Install flake8
+      - name: Install flake8, flake8-isort, and black
         run: |
           python -m pip install --upgrade pip
-          pip install flake8
+          pip install flake8 flake8-isort black
```

Comment:
Do we need this step now? I think the action below won't be using these packages, they are installed again

Reply:
I was actually planning on also adding local repo pre-commit hooks for `mypy` and `pylint` and they would require all project dependencies to be loaded since they both do some dynamic checking and imports etc.

Should I just add them to this PR? Or raise another one with those changes?